### ISSUE TITLE
Order POIs by proximity to user location

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-accessibility#3]
When the [geolocate control](https://www.mapbox.com/mapbox-gl-js/api/#geolocatecontrol) is enabled and `trackUserLocation` is enabled, the accessible elements could be ordered to start at the user location marker and radiate outward. At least this is how our iOS implementation does it. Otherwise, we could order the elements from top-left to bottom-right; either way, consistency helps the user know how far they’ve gotten through the list of elements.

/cc @tristen

**Comment by andrewharvey** [comment]
> Otherwise, we could order the elements from top-left to bottom-right; either way, consistency helps the user know how far they’ve gotten through the list of elements.

Agreed. I think top-left to bottom-right should be the default since only a small fraction of GL JS maps will have the geolocate control enabled, but also agreed that order by distance is very useful for some applications.
